src/NServiceBus.Core/Recoverability/Faults/ErrorQueueSettings.cs

```diff
@@ -11,6 +11,12 @@ namespace NServiceBus
     public static class ErrorQueueSettings
     {
         /// <summary>
+        /// The settings key where the error queue address is stored.    
+        /// </summary>
+        public const string SettingsKey = "errorQueue";
+        const string DefaultErrorQueueName = "error";
```

Comment:
We could but that made the code in https://github.com/Particular/NServiceBus/pull/4636/files/25391cbf06b57ba09fe4fe3cdc6c0627e6a74d41#diff-49b73679141b39d739de1e8bdd0d036eR37 harder to grok. Want me to change?